LAKMobile/src/screens/ListJobs.tsx

```diff
@@ -64,32 +70,34 @@ export function ListJobs({ navigation, mode }: ListJobsProps) {
 
   // Note: This should refresh every time we have a onFocus...
   useEffect(() => {
-    getUser(userId, userId).then((user) => {
-      if (user) {
-        const driverVerificationStatus = getDriverVerificationStatus(user);
-        if (driverVerificationStatus === 'Not Applied') {
-          setDriverRegistrationWarning(
-            <InfoBox
-              text={'Register as a driver to apply to jobs.'}
-              buttonText={'Register'}
-              onPress={() => navigation.navigate('DriverRegistration')}
-            />
-          );
-        } else if (
-          driverVerificationStatus === 'Applied' ||
-          driverVerificationStatus === 'In Review'
-        ) {
-          setDriverRegistrationWarning(
-            <InfoBox
-              text={
-                'Your driver registration is under review.\nYou can apply to jobs once approved.'
-              }
-            />
-          );
+    if (isFocused) {
+      getUser(userId, userId).then((user) => {
+        if (user) {
+          const driverVerificationStatus = getDriverVerificationStatus(user);
+          if (driverVerificationStatus === 'Not Applied') {
+            setDriverRegistrationWarning(
+              <InfoBox
+                text={'Register as a driver to apply to jobs.'}
+                buttonText={'Register'}
+                onPress={() => navigation.navigate('DriverRegistration')}
+              />
+            );
+          } else if (
+            driverVerificationStatus === 'Applied' ||
+            driverVerificationStatus === 'In Review'
+          ) {
+            setDriverRegistrationWarning(
+              <InfoBox
+                text={
+                  'Your driver registration is under review.\nYou can apply to jobs once approved.'
+                }
+              />
+            );
+          }
```

Comment:
Small change I added:
- We don't need to make an API call again if we're not currently focused on the screen, so wrapping this in a `if (isFocused)` will help us reduce the load on our backend since this useEffect will run every time isFocused changes, even if it changes to false.